doc/specifications/drafts/content_edit.md

```diff
@@ -0,0 +1,29 @@
+# Symfony forms content edit
+
+## Synopsis
+repository-forms currently supports creating content without an intermediate draft.
+
+Support must be added for editing content. It implies creating a new version of an existing content,
+rendering a form with the previous version's fields values, and allowing the user to save the draft,
+cancel or publish the new version. It should also allow choosing a language, either for translation
+or 
+
+Ideally, the draft would only be created when we need it, not on the first display.
```

Comment:
The warning shows up when the _user_ has a draft, does it not ? It will tell you if you already have a draft of this content, and will ask if you want to edit it, or create a new one. As far as I know, you're not warned if somebody else has a draft.

Note that this part isn't even mentioned in the requirements now. It is indeed an important part, and I'm trying to keep it in mind. Version/conflict management was only added to PlatformUI a couple weeks ago.